apps/site/lib/site_web/controllers/schedule/trip_view_controller.ex

```diff
@@ -16,6 +16,10 @@ defmodule SiteWeb.ScheduleController.TripViewController do
   plug(SiteWeb.ScheduleController.ScheduleError)
   plug(:zone_map)
 
+  def show(conn, %{"route" => "CR-" <> _ = route}) do
```

Comment:
This works, but I think to make it more future-proof, we should match on the route type instead of string matching on the name and use the `timetable_path` helper.

```
  def show(
        %{assigns: %{route: %Route{type: 2, id: route_id}}, query_params: query_params} = conn,
        _
      ) do
    redirect(conn, to: timetable_path(conn, :show, route_id, query_params))
  end
```